tools/resourcedocsgen/pkg/lookup.go

```diff
@@ -0,0 +1,196 @@
+package pkg
+
+var CategoryNameMap = map[string]PackageCategory{
+	"cloud":          PackageCategoryCloud,
+	"database":       PackageCategoryDatabase,
+	"infrastructure": PackageCategoryInfrastructure,
+	"monitoring":     PackageCategoryMonitoring,
+	"network":        PackageCategoryNetwork,
+	"utility":        PackageCategoryUtility,
+	"vcs":            PackageCategoryVCS,
+}
+
+var CategoryLookup = map[string]PackageCategory{
```

Comment:
It looks like this is what our [`keywords`](https://github.com/pulumi/pulumi/blob/578e0937a96c594cc294cdf54deaa02c0328a3e5/pkg/codegen/schema/schema.go#L1858-L1865) field does. Can we just use that?

Can we create an issue to track the follow up if it doesn't already exist?

Reply:
I'll make an issue - I'd like to keep this PR scoped to the import of `registrygen`'s functionality as much as possible.